MREUnityRuntime/MREUnityRuntimeLib/Util/UtilMethods.cs

```diff
@@ -96,5 +96,21 @@ public static void GetUrlParts(string url, out string rootUrl, out string filena
 
             filename = uri.Segments[uri.Segments.Length - 1];
         }
+
+        public static string FormatException(Exception ex)
```

Comment:
Sending a stack trace back to the app is gross. Let's just send the "unexpected error" message.

Reply:
My desired flow here is that we send back enough data for the app developer to file a github issue that's actionable. This seems like the best way for us. If the message were just "model load error", we'd have to repro the error to troubleshoot, which slows things down.

---
In reply to: [246836428](https://github.com/Microsoft/mixed-reality-extension-unity/pull/5#discussion_r246836428) [](ancestors = 246836428)